package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "realm",
   "description": "Realm is a mobile database: an alternative to SQLite and key-value stores",
-  "version": "10.17.0",
+  "version": "10.19.0-alpha.0",
```

Comment:
Should this PR be targetting the `10.19.0-alpha.0` branch?

Reply:
It it planned to be released from `master` within a week